community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/schema/LuceneDocumentStructure.java

```diff
@@ -250,6 +256,7 @@ protected AcceptStatus accept( BytesRef term ) throws IOException
         private final Field idValueField;
 
         private final Map<ValueEncoding,Field> valueFields = new EnumMap<>( ValueEncoding.class );
+        private final ArrayList<Map<ValueEncoding,Field>> compositeValueFields = new ArrayList<>();
```

Comment:
This is leftovers, but not in the way you would think, I have a hunch that it actually is compositeVauleFields that should go.